pyfolio/utils.py

```diff
@@ -117,8 +118,9 @@ def get_symbol_rets(symbol):
             if datetime.now() - pd.to_datetime(
                     getmtime(filepath), unit='s') < pd.Timedelta(days=1):
                 rets = pd.read_hdf(filepath, 'df')
-        except:
-            pass
+        except Exception as e:
```

Comment:
Also, we shouldn't be swallowing **any** exception here.  There are lots of exceptions that we don't want to catch (e.g. `MemoryError`, `KeyboardInterrupt`).  If there are specific exceptions we expect to be thrown here, we should catch them explicitly.